README.md

```diff
@@ -85,7 +85,7 @@ page content and devtools.
 ## Configuration
 Grover can be configured to adjust the layout of the resulting PDF/image.
 
-For available PDF options, see https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pagepdfoptions
+For available PDF options, see https://github.com/puppeteer/puppeteer/blob/1fbc3c643f31f529597f8ae562959b15e9baa1c5/docs/api/puppeteer.pdfoptions.md
```

Comment:
Thanks for the change @klappradla however I think it's best to reference the `main` branch rather than a specific commit, otherwise the link will become out of date pretty quickly. Balancing getting out of date vs not working at all, i know, however it's pretty unlikely that the puppeteer project will go through a major documentation refactoring again in the near future 😄 

```suggestion
For available PDF options, see https://github.com/puppeteer/puppeteer/blob/main/docs/api/puppeteer.pdfoptions.md
```

Reply:
Haha, alright 👌